VirtoCommerce.CatalogPersonalizationModule.Data/Search/SearchCriteriaExtensions.cs

```diff
@@ -16,10 +17,18 @@ public static IList<StringKeyValues> GetUserGroups(this CatalogSearchCriteriaBas
                 var nameValueDelimeter = new[] { ':' };
                 var valuesDelimeter = new[] { ',' };
 
-                result.AddRange(criteria.UserGroups
+                StringKeyValues userGroupInfo = criteria.UserGroups
```

Comment:
Why we need to parse criteria.UserGroups[]? In theory, it should contain only group values.